### PR TITLE
Windows enable credssp script - on exception Write-Error

### PR DESCRIPTION
**Pull request by bostko**

### CODE REVIEW

File: rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/CloudResource.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.apache.brooklyn.rest.resources;
+
+/**
+ * Created by valentin on 28/03/16.
+ */
+public class CloudResource {
```
Comment (neykov) [inline rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/CloudResource.java]: These couple of files don't seem related to the PR.

**Comment by neykov** [comment]
Other than that looks good.

**Comment by aledsage** [comment]
Thanks @bostko - merging now.
